zerver/tests/test_realm.py

```diff
@@ -898,6 +904,39 @@ def test_update_realm_allow_message_editing(self) -> None:
         self.assertEqual(realm.message_content_edit_limit_seconds, 200)
         self.assertEqual(realm.allow_community_topic_editing, False)
 
+    def test_prevent_realm_public_listing_if_plan_type_is_not_standard_free(self) -> None:
+        realm = get_realm("zulip")
+        realm.oss_showcase_policy = 1
+        realm.save(update_fields=["oss_showcase_policy"])
+        self.assertEqual(realm.get_plan_type, Realm.SELF_HOSTED)
+        result = self.client_patch("/json/realm", {"oss_showcase_policy": "2"})
```

Comment:
Rather than passing the values 1 and 2 here, which requires someone reading the test to look up what those values mean, instead use e.g. `str(Realm.OSS_SHOWCASE_VISIBLE)", which is much more readable.